client/Videos.js

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react";
+
+
+function Video({ data, video, setVideoData })
+{
+    const deleteFunction = (data, id) => 
+    {
+        //document.getElementById(id).remove();
+
+        fetch('http://192.168.0.15:5000/videos' + id,
+            {
+                method: 'DELETE',
+                headers: {
+                    'Content-type': 'application/json; charset=UTF-8',
+                },
+            })
+            .then((response) => response.json())
+            .then((data) => setVideoData(data))

```

Comment:
the variable name `data` is being overused in this file, leads to some confusion